src/modules/xml/producer_xml-clip.c

```diff
@@ -0,0 +1,226 @@
+/*
+ * producer_xml-native.c -- A wrapper for mlt XML in native profile
```

Comment:
This has the wrong filename in the comment header.

Reply:
done